lib/services/ios-project-service.ts

```diff
@@ -303,6 +314,12 @@ class IOSProjectService extends projectServiceBaseLib.PlatformProjectServiceBase
 		return (() => {
 			let pluginPlatformsFolderPath = pluginData.pluginPlatformsFolderPath(IOSProjectService.IOS_PLATFORM_NAME);
 			_.each(this.getAllDynamicFrameworksForPlugin(pluginData).wait(), fileName => this.addLibrary(path.join(pluginPlatformsFolderPath, fileName)).wait());
+			
+			let pluginPodFilePath = path.join(pluginPlatformsFolderPath, "Podfile");
+			if(this.$fs.exists(pluginPodFilePath).wait()) {
+				let pluginPodFileContent = this.$fs.readText(pluginPodFilePath).wait();
+				this.$fs.appendFile(this.projectPodFilePath, pluginPodFileContent).wait();
```

Comment:
Consider adding a comment in the general pod file mentioning that the code is from this given file.